frontend/src/shared.css

```
@@ -0,0 +1,158 @@
+
+@keyframes spin {
+  0%   { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+@keyframes shimmer {
+  0%, 100% { opacity: 0.6; }
+  50%       { opacity: 1; }
+}
+
+@keyframes errorSlideIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "errorSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)